kt/plugins/godot-intellij-plugin/build.gradle.kts

```diff
@@ -1,68 +1,53 @@
-import godot.dependencies.gradle.helper.BuildConfig
-import godot.dependencies.gradle.helper.VersionRange
-import godot.dependencies.gradle.fullGodotKotlinJvmVersion
-import godot.dependencies.gradle.isSnapshot
+import versioninfo.intellij.BuildConfig
+import versioninfo.intellij.VersionRange
+import versioninfo.fullGodotKotlinJvmVersion
+import versioninfo.isSnapshot
+import org.jetbrains.changelog.Changelog
 
 plugins {
     // Java support
     id("java")
     // Kotlin support
-    id("org.jetbrains.kotlin.jvm")
+    alias(libs.plugins.kotlin.jvm)
     // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
-    id("org.jetbrains.intellij") version "1.13.2"
+    alias(libs.plugins.gradleIntelliJPlugin)
     // gradle-changelog-plugin - read more: https://github.com/JetBrains/gradle-changelog-plugin
-    id("org.jetbrains.changelog") version "1.3.1"
-    id("com.utopia-rise.godot-dependencies")
+    alias(libs.plugins.changelog)
+
+    id("com.utopia-rise.versioninfo")
 }
 
 //sdk version: https://github.com/JetBrains/intellij-community/tags
 //kotlin plugin version: https://plugins.jetbrains.com/plugin/6954-kotlin/versions
 val buildMatrix: Map<String, BuildConfig> = mapOf(
-    "IJ213" to BuildConfig(
-        sdk = "213.6777.52",
-        prefix = "IJ2021.3",
-        extraSource = "IJ213",
-        version = VersionRange("213.2", "213.*"),
-        ideVersionsForVerifierTask = listOf("2021.3"),
-        deps = listOf("java", "org.jetbrains.kotlin:213-1.6.21-release-334-IJ6777.52", "gradle")
-    ),
-    "IJ221" to BuildConfig(
-        sdk = "221.5591.52",
-        prefix = "IJ2022.1",
-        extraSource = "IJ221",
-        version = VersionRange("221.3", "999.*"),
-        ideVersionsForVerifierTask = listOf("2022.1"),
-        deps = listOf("java", "org.jetbrains.kotlin:221-1.7.10-release-333-IJ5591.52", "gradle")
-    ),
-    "IJ222" to BuildConfig(
-        sdk = "222.3345.118",
-        prefix = "IJ2022.2",
-        extraSource = "IJ221", // hasn't changed. Thus no need to update
-        version = VersionRange("222.2", "999.*"),
-        ideVersionsForVerifierTask = listOf("2022.2"),
-        deps = listOf("java", "org.jetbrains.kotlin", "gradle") // kotlin plugin version no longer needed as it's now bundled with the IDE
+    "IJ232" to BuildConfig(
+        sdk = "232.8660.185",
+        prefix = "IJ2023.2",
+        extraSource = "",
+        version = VersionRange("232.1", "999.*"),
+        ideVersionsForVerifierTask = listOf("2023.2"),
+        deps = listOf(
+            "java",
+            "org.jetbrains.kotlin",
+            "gradle"
+        ) // kotlin plugin version no longer needed as it's now bundled with the IDE
     )
 )
 
+// needed as the intellij plugin does add its own repositories and thus the ones defined in the settings.gradle.kts are ignored. Hence, we need to redefine them here
 repositories {
+    mavenLocal()
     mavenCentral()
-    maven {
```

Comment:
Why are those removed ?

Reply:
These are now automatically added by https://github.com/utopia-rise/godot-kotlin-jvm/blob/0a7c78e3b2bd64e4a99cc64d302bc09ce9605236/kt/plugins/godot-intellij-plugin/build.gradle.kts#L14
Also the reason of: https://github.com/utopia-rise/godot-kotlin-jvm/blob/0a7c78e3b2bd64e4a99cc64d302bc09ce9605236/kt/plugins/godot-intellij-plugin/build.gradle.kts#L38